Developers/Unity Project/Spooksville/Assets/Scripts/Player/PlayerMovement.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerMovement : MonoBehaviour
+{
+    public string[] maskedLayers;
+    public float speed = 1.0f;
+
+    private BoxCollider2D boxCollider;
+    private Vector3 direction;
+    private RaycastHit2D hit;
+
+    void Start()
+    {
+        direction = Vector3.zero;
+        boxCollider = GetComponent<BoxCollider2D>();
+    }
+
+    void FixedUpdate()
+    {
+        float x = Input.GetAxisRaw("Horizontal");
```

Comment:
In general I'd encourage the use of type inference wherever possible - keeps the code more terse and makes it easier to change. Everything still has a static type, you just don't have to bother writing it out.
```suggestion
        var x = Input.GetAxisRaw("Horizontal");
```